### PR TITLE
hystrix: 1.3.20 → 1.5.18

### PR DESCRIPTION
**Pull request by magsel**
cleaner this time around

### CODE REVIEW

File: src/main/java/com/signicat/hystrix/servlet/ConcurrencyStrategyWithCoreTimeOut.java [hunk 1]
```diff
@@ -16,25 +17,20 @@
 import java.util.concurrent.TimeUnit;
 
 /**
- * Concurrency strategy which specifies an explicit core pool size. This is needed
```
Comment (einarmr) [inline src/main/java/com/signicat/hystrix/servlet/ConcurrencyStrategyWithCoreTimeOut.java]: Have you check if this comment still holds? If it does, your change won't do any good.

Comment (magsel) [reply src/main/java/com/signicat/hystrix/servlet/ConcurrencyStrategyWithCoreTimeOut.java]: yeah, the thing is, HystrixServlet now provides diverging core thread size and max thread size, look in `AsyncWrapperServlet`, line 141

File: src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java [hunk 2]
```diff
@@ -117,7 +117,7 @@ public void service(final HttpServletRequest req, final HttpServletResponse resp
 
         //double thread pool size for pool with magic name 'default'
         final int maximumSize = DEFAULT_COMMAND_GROUP_KEY.equals(key) ? (corePoolSize * 2) : corePoolSize;
-        final int coreSize    = Math.min(10, maximumSize);
+        final int coreSize    = Math.max(10, maximumSize);
```
Comment (einarmr) [inline src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java]: max? really?

File: src/main/java/com/signicat/hystrix/servlet/ConcurrencyStrategyWithCoreTimeOut.java (hunk 1 above)
Comment (einarmr) [reply src/main/java/com/signicat/hystrix/servlet/ConcurrencyStrategyWithCoreTimeOut.java]: Perfect

File: src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java (hunk 2 above)
Comment (magsel) [reply src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java]: yeah, pretty sure, i agree it's strange. i noticed portal behaving differently, and had another look at the original code.

Comment (einarmr) [reply src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java]: This is easily unit testable for various values

Comment (magsel) [reply src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java]: yeah, please ignore that, i was just seeing double for a moment

File: src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java [hunk 3]
```diff
@@ -116,8 +116,8 @@ public void service(final HttpServletRequest req, final HttpServletResponse resp
         log.info("Scheduling Hystrix command with key '" + key + "' for path: '" + path + "'.");
 
         //double thread pool size for pool with magic name 'default'
-        final int size = DEFAULT_COMMAND_GROUP_KEY.equals(key) ? (corePoolSize * 2) : corePoolSize;
-        final int queueSize = (int) (((double) corePoolSize) * 1.4d);
+        final int coreSize = DEFAULT_COMMAND_GROUP_KEY.equals(key) ? (corePoolSize * 2) : corePoolSize;
```
Comment (einarmr) [inline src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java]: Might as well revert this properly (rename the variable again, and fix whitespace on line 120)

Comment (axelkvistad) [reply src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java]: Personally I think this alignment makes for better readability. No accounting for taste though, right? :)

Comment (einarmr) [reply src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java]: IMHO less noise in pull requests is always preferred. We want to see what you wanted to change, not what just changed due to personal taste or accidents.

Comment (axelkvistad) [reply src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java]: IMO there's a difference between indenting a single line and reformatting in a way that obfuscates functional changes. We're contributing more (still negligible) noise just by having this discussion :P

Comment (einarmr) [reply src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java]: No, we're not. "git blame" will return the wrong thing by merging this change. Long-term, you would want "git blame" to return something useful.

Comment (axelkvistad) [reply src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java]: Fair enough. I agree with everything you say, but I think you're making it out to be a bigger problem than it really is. Nitpicking over miniscule issues is generally not all that constructive (perfect is the enemy of good etc.). Feel free to disagree.

**Pull request merged**